### PR TITLE
MRG: Fix links

### PR DESCRIPTION
**Pull request by larsoner**
Just some minor tweaks to fix `make linkcheck`:

https://circleci.com/gh/mne-tools/mne-python/11092

### CODE REVIEW

File: tutorials/plot_sensors_decoding.py [hunk 1]
```diff
@@ -406,7 +406,7 @@
 # ==========
 # .. [1] Jean-Rémi King et al. (2018) "Encoding and Decoding Neuronal Dynamics:
 #        Methodological Framework to Uncover the Algorithms of Cognition",
-#        in press. https://hal.archives-ouvertes.fr/hal-01848442/
```
Comment (agramfort) [inline tutorials/plot_sensors_decoding.py]: why removing this link?

Comment (larsoner) [reply tutorials/plot_sensors_decoding.py]: It timed out on CircleCI, and took a very long time to load when I tried, so it seemed unreliable. Want me to put it back?

Comment (larsoner) [reply tutorials/plot_sensors_decoding.py]: I put it back with a new entry in `linkcheck_ignore`

**Comment by agramfort** [comment]
thx. Please merge if appveyor error is unrelated.

**Pull request merged**